modules/4337/contracts/experimental/P256Wrapper.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0 <0.9.0;
+
+/**
+ * @title P256Wrapper contract
+ * @dev A contract that implements P256 wrapper with mallability check.
+ *      As recommended in https://eips.ethereum.org/EIPS/eip-7212
+ *      Follows precompile interface.
+ * @custom:acknowledgement The contract is heavily inspired by https://github.com/daimo-eth/p256-verifier
+ */
+abstract contract P256Wrapper {
+    address immutable VERIFIER;
+
+    constructor(address verifier) {
+        require(verifier != address(0), "P256: invalid verifier address");
+        VERIFIER = verifier;
+    }
+
+    /// P256 curve order n/2 for malleability check
```

Comment:
uber-OCD-nit: I would use consistent comment format for all fields, here it has `///`, but in other places it has `/** */`.

Reply:
I personally like to stick to `//` for regular comments and `/** */` for multiline. How does that sound?